.clang-tidy

```diff
@@ -246,7 +247,6 @@ Checks: >
   performance-noexcept-destructor,
   performance-noexcept-swap,
   portability-avoid-pragma-once,
```

Comment:
```suggestion
```

Imho we should remove avoidance of pragme-once as well.
It wasnt too long ago that we went from header guard to pragma once, it would be a shame to go back. All compilers that we support do support pragma once